64.py

```diff
@@ -0,0 +1,17 @@
+class Solution:
+    def minPathSum(self, grid: List[List[int]]) -> int:
+        m, n = len(grid), len(grid[0])
```

Comment:
I'd like to use all upper case letters to name lengths of arrays or lists, because they are constants.